src/cascade/model/compartmental.py

```diff
@@ -9,8 +9,8 @@
 
 1.  Specify a disease process by making simple Python functions that
     return disease rates as a function of time. You can specify a set
-    of :math:`(\iota, \rho, \chi, \mu)`, or you can specify a set
-    of :math:`(\iota, \rho, \chi, \omega)`. We'll call the former the
+    of :math:`(\iota, \\rho, \chi, \mu)`, or you can specify a set
+    of :math:`(\iota, \\rho, \chi, \omega)`. We'll call the former the
```

Comment:
What is the double backslash adding?

Reply:
It adds success! Without it sphinx fails because `\r` is a special thing so you have to escape the slash in order to get `\rho`

Reply:
Oh, I forgot about raw strings. That is absolutely the right way to do this.